scripts/tests/backwards-compatibility-test.sh

```diff
@@ -2,8 +2,11 @@
 
 set -euo pipefail
 
+# versions to test when run without arguments
+default_versions=("v0.2.1")
```

Comment:
Maybe just make it always parametric without a default?

Reply:
I like having the default since it allows us to invoke the script in CI without parameters, i.e. one less area we may forget to update in the future.

https://github.com/fedimint/fedimint/blob/6d20243/.github/workflows/ci-backwards-compatibility.yml#L48

I don't have a strong opinion though, so we can rip and instead explicitly pass.